packages/duckdb/meta.yaml

```diff
@@ -1,13 +1,13 @@
 package:
   name: duckdb
-  version: 1.1.2
+  version: 1.2.0
   # TODO: re-enable once we have a build for abi 2025_0
   _disabled: true
```

Comment:
Currently, we are in the process of upgrading the Emscripten version that Pyodide uses (#5332), which introduces breaking changes in ABI. Therefore, we temporarily disabled duckdb. For now, you can use the [nightly toolchain](https://github.com/pyodide/pyodide-build-environment-nightly) to build duckdb against the latest Pyodide, but we will soon update the Emscripten version again.

Reply:
Ok, thanks!

It's not 100% clear to me what I should do with this PR then. Should I wait for the emscripten bump to happen and then make a PR to duckdb to start building against that version?

Reply:
> make a PR to duckdb to start building against the new emscripten version?

Yes, I would suggest you to do this. I don't think it would take very long to upgrade the Emscripten version.

Reply:
> I don't think it would take very long to upgrade the Emscripten version.

It might not be that quick, and we need to make more Emscripten updates after that. @cpcloud if you could build a version with Emscripten 3.1.74 (what we have currently) and we can check that CI passes, then we could land this. I'd disable it again when we update Emscripten again and then when the Emscripten updates are done you can do a second build with the newest version and land that.